bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:sncf:api">
+		<parameter name="apiID" type="text">
+			<label>API ID</label>
+			<required>true</required>
```

Comment:
This tag is now deprecated. Instead please add a "required" attribute to the parameter tag.